otter/integration/tests/test_lbsh.py

```diff
@@ -120,24 +119,20 @@ def test_move_node_to_oob_lb(self):
 
         clb_as = self.helper.clbs[0]
 
-        nodes_as = yield clb_as.list_nodes(self.rcs)
-        nodes_other = yield clb_other.list_nodes(self.rcs)
-        self.assertEqual(len(nodes_as['nodes']), 0,
-                         "There should be no nodes on the CLB yet.")
-        self.assertEqual(len(nodes_other['nodes']), 0,
-                         "There should be no nodes on the CLB yet.")
+        yield clb_as.wait_for_nodes(
+            self.rcs, HasLength(0), timeout=timeout_default)
+        yield clb_other.wait_for_nodes(
+            self.rcs, HasLength(0), timeout=timeout_default)
 
         group, _ = self.helper.create_group(min_entities=1)
         yield self.helper.start_group_and_wait(group, self.rcs)
 
         nodes_as = yield clb_as.list_nodes(self.rcs)
```

Comment:
This should probably be moved down below the `wait_for_nodes` calls, since listing might result in 0 nodes if CLB hasn't updated yet.  If it's done after the waits, then we can be sure that there is 1.